Project1/Tile.cpp

```diff
@@ -20,3 +23,8 @@ bool Tile::hasUnit() const{
 	return _unit != nullptr;
 }
 
+bool Tile::operator==(const Tile & tile) const
+{
+	return this->_terrain == tile._terrain && this->_unit == tile._unit;
+}
```

Comment:
Do you think it would make more or less sense for this to be
`return this == &tile;`

Reply:
true